Header header
uint8 pixel_1
uint8 pixel_2
uint8 pixel_3
uint8 pixel_4
